en/controllers.rst

```diff
@@ -670,8 +670,8 @@ Other Useful Methods
             array('limit' => 5, 'order' => 'Article.created DESC')
         );
 
-        $this->loadModel('User', 2);
-        $user = $this->User->read();
+        $this->loadModel('User');
```

Comment:
In that case we also can remove those two lines in each language because it's pretty much the same as the example above or at least it adds nothing worth keeping.